test/integration/access_log_integration_test.cc

```diff
@@ -0,0 +1,184 @@
+#include "common/buffer/zero_copy_input_stream_impl.h"
+#include "common/common/version.h"
+#include "common/grpc/codec.h"
+#include "common/grpc/common.h"
+
+#include "test/integration/http_integration.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace {
+
+class AccessLogIntegrationTest : public HttpIntegrationTest,
+                                 public testing::TestWithParam<Network::Address::IpVersion> {
+public:
+  AccessLogIntegrationTest() : HttpIntegrationTest(Http::CodecClient::Type::HTTP1, GetParam()) {}
+
+  void createUpstreams() override {
+    HttpIntegrationTest::createUpstreams();
+    fake_upstreams_.emplace_back(new FakeUpstream(0, FakeHttpConnection::Type::HTTP2, version_));
+  }
+
+  void initialize() override {
+    config_helper_.addConfigModifier([](envoy::api::v2::Bootstrap& bootstrap) {
+      bootstrap.mutable_rate_limit_service()->set_cluster_name("accesslog");
```

Comment:
curious: Why is the ratelimit service being used here?

Reply:
bug/copy+paste